package.json

```diff
@@ -5,7 +5,10 @@
   "scripts": {
     "postcheckout": "yarn",
     "start": "node ./bin/www",
-    "dev": " concurrently --kill-others \"mongod\" \"grunt watch\" \"nodemon ./bin/www\" ",
+    "watch": "nodemon ./bin/www --ignore public/",
+    "dev":
```

Comment:
Should `mongod` still be added here? Or is it better practice to have the developer run mongod separately? Also how about the `--kill-others` command?

Reply:
I haven't really found any other projects that start mongo in package.json and it does clutter the output quite a bit so I was thinking we could cut it.

Reply:
We could add `--kill-others` back though!